src/coreclr/jit/hwintrinsiclistxarch.h

```diff
@@ -675,7 +675,7 @@ HARDWARE_INTRINSIC(SSE42_X64,       Crc32,
 //                                                                                                      {TYP_BYTE,              TYP_UBYTE,              TYP_SHORT,              TYP_USHORT,             TYP_INT,                TYP_UINT,               TYP_LONG,               TYP_ULONG,              TYP_FLOAT,              TYP_DOUBLE}
 // ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
 //  AVX Intrinsics
-HARDWARE_INTRINSIC(AVX,             Add,                                        32,              2,     {INS_invalid,           INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_addps,              INS_addpd},             HW_Category_SimpleSIMD,             HW_Flag_Commutative)
```

Comment:
Would be good to also cover `SSE_Add`, `SSE2_Add`, `AVX512F_Add`, and `AVX512BW_Add` simultaneously. That way we can have higher confidence that the implementation is correct/working for all 3 vector sizes.

Reply:
We will include support for those as well, as we will include embedded broadcast support for all 3 types of vector length in this PR.